PrebidMobile/API1.0/src/main/java/org/prebid/mobile/NativeAdUnit.java

```diff
@@ -0,0 +1,197 @@
+package org.prebid.mobile;
+
+import android.support.annotation.NonNull;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * For details of the configuration of native imps, please check this documentation:
+ * https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf
+ */
+public class NativeAdUnit extends AdUnit {
```

Comment:
1. Magic numbers
I propose to create a documented, predefined named constant variables for a public API that will describe a destination of an argument

For example the `addEventTracker` method has `Integer event` and it is not clear what exactly it means as a result it is simple to make a mistake. 

More over the implementation can be 
``` Java
public static final class EventType {

    static final int IMPRESSION_ID = 1;
    static final int VIEWABLE_MRC50_ID = 2;
    //...

    /**
    * Impression
    */
    public static final EventType IMPRESSION = new EventType(IMPRESSION_ID);

    /**
    * Visible impression using	MRC	definition
    * at 50% in view for 1	second
    */
    public static final EventType VIEWABLE_MRC50 = new EventType(VIEWABLE_MRC50_ID);

    private final int id;

    public EventType(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }
}

```

and usage

``` Java
adUnit.addEventTracker(EventType.IMPRESSION, ...);
adUnit.addEventTracker(new NativeAdUnit.EventType(500), ...);

```

Reply:
It is can not be an enum because usually a type can contain a user defined value. For example `7.6 Event Types Table` has `500+ exchange-specific` as a result a value can be 500, 501, 502...

I think that we should create separate classes or just a predefined named constants for every type(e.g. `EventType`, `Image Type`, `Data Type`, `mimes`,  `methods`...)